internal/cmd/egctl/config.go

```diff
@@ -418,7 +418,7 @@ func extractConfigDump(fw kube.PortForwarder) (*adminv3.ConfigDump, error) {
 }
 
 func configDumpRequest(address string) ([]byte, error) {
-	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/config_dump", address), nil)
+	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/config_dump?include_eds", address), nil)
```

Comment:
include_eds is expensive, what about just request when you're retrieving endpoint?

Reply:
reference this
 https://www.envoyproxy.io/docs/envoy/latest/operations/admin.html#get--config_dump?include_eds

Reply:
I will optimize this by ep retrieve